### ISSUE TITLE
[Question] Use for multiple organs?

### ISSUE THREAD

**Issue by ajv012** [gevaertlab/NucSegAI#1]
Thank you for developing and open-sourcing your tools! Have your tools been primarily trained and evaluated on a single organ, or have you found them to generalize well across multiple organs? I am curious to hear about your experience applying these tools to different organ types.

**Comment by YuanningEric** [comment]
Hi, thanks for your interest! Our models were initially pre-trained on images from colorectal cancer and then fine-tuned using data from non-small-cell lung cancer (NSCLC). As a result, we currently provide two checkpoints: (1) NucSegAI_CRC for colorectal cancer and (2) NucSegAI_NSCLC for lung cancer.

We have tested the NSCLC model on breast cancer images, but observed reduced performance. This suggests that model accuracy is indeed influenced by cancer type. We plan to expand the model to additional cancer types as more multi-modal datasets become available.
